platform/darwin/scripts/style-spec-overrides-v8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "layout_symbol": {
+    "icon-text-fit-padding": {
+      "doc": "Size of the additional area added to dimensions determined by `icon-text-fit`, in counterclockwise order: top, left, bottom, right."
```

Comment:
How about:

> Padding added to the dimensions determined by the `iconTextFit` property.

We already mention in the generated documentation that the NSValue contains an NSEdgeInsets or UIEdgeInsets. The developer can look up what the expected order is for those types.